<launch>
  <arg name="grasp_topic" default="grasp" />
  <arg name="calculated_poses_topic" default="grasp" />

  <node pkg="remote_manipulation_markers" type="point_and_click" name="point_and_click">
    <param name="grasp_topic" value="$(arg grasp_topic)" />
    <param name="calculated_poses_topic" value="$(arg calculated_poses_topic)" />
  </node>
</launch>